kolibri/plugins/style_guide/templates/style_guide/style_guide.html

```diff
@@ -1,3 +1,5 @@
+<link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
```

Comment:
Ah, ok. I didn't know the Material Design fonts are already installed. So I've changed my PR to use that instead. Please take another look.